plugins/Toolbox/src/Toolbox.py

```diff
@@ -377,7 +371,6 @@ def resetMaterialsQualitiesAndUninstall(self) -> None:
     def onLicenseAccepted(self):
         self.closeLicenseDialog.emit()
         package_id = self.install(self.getLicenseDialogPluginFileLocation())
-        self.subscribe(package_id)
```

Comment:
Hmm, I'd still expect it to subscribe also if you install it via the Marketplace? I'll test if this is still working. Seems that you intended to remove this, so I don't understand. Was this removed just for testing maybe?